pkgs/applications/virtualization/xen/4.8.nix

```diff
@@ -43,18 +43,23 @@ let
     sha256 = "0gaz93kb33qc0jx6iphvny0yrd17i8zhcl3a9ky5ylc2idz0wiwa";
   };
 
+  # Ported from
+  #"https://xenbits.xen.org/gitweb/?p=qemu-xen.git;a=patch;h=e014dbe74e0484188164c61ff6843f8a04a8cb9d";
+  #"https://xenbits.xen.org/gitweb/?p=qemu-xen.git;a=patch;h=0e3b891fefacc0e49f3c8ffa3a753b69eb7214d2";
+  qemuGlusterfs6Fix = ./qemu-gluster-6-compat.diff;
```

Comment:
nixpkgs policy generally is to not include petches if we can fetchpatch them. Can you change this?

Reply:
The patches don’t apply cleanly (xen 4.8.5 is from 2017 and patches are from 2019). Had to fix manually

Reply:
Fair enough then.